lib/mediaType.js

```diff
@@ -99,7 +99,7 @@ function specify(type, spec) {
 
   return {
     q: spec.q,
-    i: spec.i,
+    i: order,
```

Comment:
I'm gonna write a few more tests and see how to keep both the client order (if present) and the server order ...